lib/src/marker_cluster_layer_options.dart

```diff
@@ -38,18 +37,6 @@ class AnimationsOptions {
   });
 }
 
-class PopupOptions {
```

Comment:
This seems to be out of scope of this PR, maybe move it out into a separate one.